helm/trino-gateway/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "8"
+version: "10.0.0"
```

Comment:
In our prior syncs we decided to keep Trino Gateway and helm chart versions in sync for now. So at this stage it should be just "10".

Reply:
But we can also discuss in the dev sync on Wednesday